bin/post-flight

```diff
@@ -43,8 +43,8 @@ EOF
   fi
 }
 
-__git_branch() {
-  git rev-parse --abbrev-ref HEAD
+__git_branch_at_commit() {
+  printf "%s @ %s" "$(git rev-parse --abbrev-ref HEAD)" "$(git log --oneline -1)"
```

Comment:
This makes for a pretty long line, at least for me.  Can we instead do `git log --format='%h' -1`?